go.mod

```diff
@@ -3,21 +3,24 @@ module github.com/giantswarm/management-cluster-admission
 go 1.16
 
 require (
+	github.com/form3tech-oss/jwt-go v3.2.2+incompatible // indirect
 	github.com/giantswarm/apiextensions/v3 v3.22.0
 	github.com/giantswarm/microerror v0.3.0
 	github.com/go-logr/zapr v0.2.0
 	github.com/gogo/protobuf v1.3.2 // indirect
 	github.com/spf13/pflag v1.0.5
 	go.uber.org/zap v1.16.0
-	k8s.io/api v0.20.2
-	k8s.io/apimachinery v0.20.2
-	k8s.io/client-go v0.20.2
+	k8s.io/api v0.18.9
+	k8s.io/apimachinery v0.18.9
+	k8s.io/client-go v0.18.9
 	sigs.k8s.io/cluster-api v0.3.13
-	sigs.k8s.io/controller-runtime v0.8.3
+	sigs.k8s.io/controller-runtime v0.6.3
 )
 
 replace (
 	github.com/coreos/etcd => github.com/coreos/etcd v3.3.24+incompatible
-	github.com/dgrijalva/jwt-go => github.com/dgrijalva/jwt-go/v4 v4.0.0-preview1
+	github.com/dgrijalva/jwt-go => github.com/form3tech-oss/jwt-go v3.2.1+incompatible
+	github.com/go-logr/logr v0.2.0 => github.com/go-logr/logr v0.2.1
 	github.com/gorilla/websocket => github.com/gorilla/websocket v1.4.2
+	sigs.k8s.io/cluster-api => github.com/giantswarm/cluster-api v0.3.13-gs
```

Comment:
Unfortunately, to get this to compile, I had to do some caveman shit. I was running into these errors, and this is the only dependency combination I found that worked.
```
# sigs.k8s.io/cluster-api/util/predicates
../../../../../.gvm/pkgsets/go1.16/global/pkg/mod/sigs.k8s.io/cluster-api@v0.3.16/util/predicates/generic_predicates.go:147:91: e.MetaNew undefined (type event.UpdateEvent has no field or method MetaNew)
../../../../../.gvm/pkgsets/go1.16/global/pkg/mod/sigs.k8s.io/cluster-api@v0.3.16/util/predicates/generic_predicates.go:150:88: e.Meta undefined (type event.CreateEvent has no field or method Meta)
../../../../../.gvm/pkgsets/go1.16/global/pkg/mod/sigs.k8s.io/cluster-api@v0.3.16/util/predicates/generic_predicates.go:153:88: e.Meta undefined (type event.DeleteEvent has no field or method Meta)
../../../../../.gvm/pkgsets/go1.16/global/pkg/mod/sigs.k8s.io/cluster-api@v0.3.16/util/predicates/generic_predicates.go:156:89: e.Meta undefined (type event.GenericEvent has no field or method Meta)
```